Aquality.Selenium.Core/src/Aquality.Selenium.Core/Elements/Interfaces/IElementFactory.cs

```diff
@@ -33,10 +33,24 @@ public interface IElementFactory
         /// <returns>Instance of child element</returns>
         T FindChildElement<T>(IElement parentElement, By childLocator, string name = null, ElementSupplier<T> supplier = null, ElementState state = ElementState.Displayed) where T : IElement;
 
+        /// <summary>
+        /// Finds list of child elements by their locator relative to parent element.
+        /// </summary>
+        /// <typeparam name="T">Type of child elements that has to implement IElement.</typeparam>
+        /// <param name="parentElement">Parent element.</param>
+        /// <param name="childLocator">Locator of child elements relative to their parent.</param>
+        /// <param name="supplier">Delegate that defines constructor of element in case of custom element type.</param>
+        /// <param name="expectedCount">Expected number of elements that have to be found (zero, more then zero, any).</param>
+        /// <param name="state">Child elements state.</param>
+        /// <param name="name">Child elements name.</param>
+        /// <exception cref="InvalidOperationException">Thrown when the supplier is null, and no constructor with required arguments was found.</exception>
+        /// <returns>List of child elements.</returns>
+        IList<T> FindChildElements<T>(IElement parentElement, By childLocator, string name = null, ElementSupplier<T> supplier = null, ElementsCount expectedCount = ElementsCount.Any, ElementState state = ElementState.Displayed) where T : IElement;
```

Comment:
Does it make sense to change default value of state to ExistsInAnyState?

Reply:
Currently `ElementState state = ElementState.Displayed` is a default value of all the methods in the current interface. 
Do you mind to change them all?
If so, it would be a breaking change I believe :)

Reply:
Ok, will discuss it